src/002_installation.md

```diff
@@ -55,48 +55,49 @@ Continue as described in Section [Running Tamarin](#sec:running-tamarin) to run
 Mac OS X {#sec:macosx}
 --------
 
-### Automatic Installation of the Tamarin binary {#sec:MacOSBinInstall}
+### Automatic Installation of Tamarin {#sec:MacOSBinInstall}
 
-The fastest way to install Tamarin on Mac OS X is to use [Homebrew](http://brew.sh/) [Cask](https://caskroom.github.io/).
+The fastest way to install Tamarin on Mac OS X is to use [Homebrew](http://brew.sh/).
 
 If you already have this installed, it is as simple as running the following two commands in your terminal:
 
   * `brew update`
-  * `brew cask install tamarin-prover`
+  * `brew install homebrew/science/tamarin-prover`
```

Comment:
Happy to change it to `brew install tamarin-prover` if that works from a fresh install of Homebrew. I basically want there to be as few steps as possible: if tapping Homebrew Science is necessary, then I'd rather users are asked to do that in the `brew install homebrew/science/tamarin-prover` command, rather than a separate `brew tap homebrew/science`. 

Obviously the simple `brew install tamarin-prover` is the ideal case.